src/escapedHashtags.js

```diff
@@ -32,6 +32,7 @@ define([
     var OUTSIDE_HASHTAG = 0,
         INSIDE_HASHTAG = 1,
         DELIMITER = "`",
+        ID_CHAR = /^[\w.-]/,
```

Comment:
Picky, but how about escaping the hyphen? It's easy to edit regexes with `[]` and not remember that the hyphen needs to go last if it isn't escaped.